src/components/Modal_Components/dialogViews/dialogView2.js

```diff
@@ -1,14 +1,16 @@
 import React from "react";
 import {
-  Dialog,
-  ModalTitle,
-  DialogContent,
   Select,
   FormControl,
   TextField,
-  DialogActions,
-  Button
 } from "./dialogComponents";
+import ModalDialog from '../ModalDialog'
+
+/**
+ * View2 
+ * @param {*} param0 props{
+ * }
+ */
```

Comment:
We need to refactor this jsDoc to follow whatever the best practice is for React components